tools/azure-sdk-tools/packaging_tools/auto_codegen.py

```diff
@@ -77,6 +131,9 @@ def main(generate_input, generate_output):
             # Generate some necessary file for new service
             init_new_service(package_name, folder_name)
 
+            # Update metadata
+            update_service_metadata(sdk_folder, data, config, folder_name, package_name, spec_folder, input_readme)
```

Comment:
If there is any exception thrown from `update_service_metadata`, we should not block the main process.